public/js/main.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(() => {
```

Comment:
you should keep your `$(document).ready(() => {})` as small as possible, separate out your function and then just include the code that needs to execute when the page loads in `$(document).ready()`